### PR TITLE
DEP Allow symfony 4

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripeltd/product-issues/issues/492

### CODE REVIEW

**Comment by maxime-rainville** [comment]
The builds were saying something weird about not finding the correct hash for graphql so I restarted them just to double check if it's an intermittent failure.

**Comment by emteknetnz** [comment]
There was a yml string that needed quoting, have updated

**Comment by MarcEspiard** [comment]
Hi, can this be merged and released please, it's blocking the install of this package on SS 4.8

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
         "silverstripe/framework": "^4.10",
         "silverstripe/admin": "^1",
         "silverstripe/versioned": "^1",
-        "symfony/yaml": "~3.2"
+        "symfony/yaml": "~3.2 || ^4"
```
Comment (michalkleiner) [inline composer.json]: Can we do this? Not that it makes much difference, but would be closer to all the other caret constraints.
```suggestion
        "symfony/yaml": "^3 || ^4"
```

**Review by michalkleiner — APPROVED** [review]

**Review by maxime-rainville — APPROVED** [review]
LGTM

**Comment by maxime-rainville** [comment]
@MarcEspiard This shouldn't be blocking installation on 4.8. Maybe force your 4.8 project to install `symfony/yaml` 3. If that doesn't fix it, I can tag 5.6.0-beta2 so you have a tag you can target.

**Pull request merged**